HermesProxy/App.config

```diff
@@ -16,7 +16,7 @@
     <!--
             Option:      ServerBuild
             Description: The version of the server you want to connect to.
-            Default:     "5875" (1.12.1)
+            Default:     "5875" (1.12.1) "8606" (2.4.3)
```

Comment:
put it on a new line

Reply:
Like this?
            Default:     "40618" (1.14.0)
                             "40892" (2.5.2)